src/Pim/Bundle/ApiBundle/tests/integration/Controller/Product/DeleteProductIntegration.php

```diff
@@ -23,16 +23,15 @@ public function testDeleteAProduct()
         $client = $this->createAuthenticatedClient();
 
         $this->assertCount(3, $this->get('pim_catalog.repository.product')->findAll());
-        // TODO: TIP-613: This repository does not exist anymore. We need to find another way to count.
-        $this->assertEquals(30, $this->get('pim_catalog.repository.product_value_counter')->count());
+        $this->assertProductRawValuesEquals(27);
```

Comment:
not really relevant for the API, bu we wanted to check if product values are really deleted when you delete a product. should be done on a low level test (on Remover for instance)